packages/webpack-common/src/extractText.js

```diff
@@ -18,7 +18,7 @@ export {
  */
 function getLoaderConfigByType (context, webpackConfig, fileType) {
   const loaderConfig = webpackConfig.module.loaders.find(
-    (loader) => loader.test === context.fileType(fileType)
+    (loader) => regexEqual(loader.test, context.fileType(fileType))
```

Comment:
It is unfortunally not the case anymore when I updated webpack-merge to 2.0.0. A test was failing because it couldn't find the correct loader for text/css.